src/backend/drm/legacy/mod.rs

```diff
@@ -273,6 +233,9 @@ impl<A: AsRawFd + 'static> Device for LegacyDrmDevice<A> {
             }
         }
 
+        // Now try to do the least possible amount of changes to set this to the state the users requested
+        // TODO!
```

Comment:
Does this `TODO` means that the current code is simply not as efficient as it could be, or is it something that needs to be fixed for merge?

Reply:
This should be a left-over, I have implemented this differently. Good catch though, I will clean this up.